packages/runtime/container-runtime/src/blobManager/blobManager.ts

```diff
@@ -936,22 +845,28 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 }
 
 /**
- * For a blobId, returns its path in GC's graph. The node path is of the format `/<BlobManager.basePath>/<blobId>`.
+ * For a blobId, returns its path in GC's graph. The node path is of the format `/<blobManagerBasePath>/<blobId>`.
  * This path must match the path of the blob handle returned by the createBlob API because blobs are marked
  * referenced by storing these handles in a referenced DDS.
  */
-function getGCNodePathFromBlobId(blobId: string) {
-	return `/${BlobManager.basePath}/${blobId}`;
-}
+const getGCNodePathFromBlobId = (blobId: string) => `/${blobManagerBasePath}/${blobId}`;
 
 /**
- * For a given GC node path, return the blobId. The node path is of the format `/<BlobManager.basePath>/<blobId>`.
+ * For a given GC node path, return the blobId. The node path is of the format `/<basePath>/<blobId>`.
  */
-function getBlobIdFromGCNodePath(nodePath: string) {
+const getBlobIdFromGCNodePath = (nodePath: string) => {
 	const pathParts = nodePath.split("/");
-	assert(
-		pathParts.length === 3 && pathParts[1] === BlobManager.basePath,
-		0x5bd /* Invalid blob node path */,
-	);
+	assert(areBlobPathParts(pathParts), 0x5bd /* Invalid blob node path */);
 	return pathParts[2];
-}
+};
+
+/**
+ * Returns whether a given path is for attachment blobs that are in the format - "/blobManagerBasePath/...".
+ */
+export const isBlobPath = (path: string): path is `/${typeof blobManagerBasePath}/${string}` =>
+	areBlobPathParts(path.split("/"));
+
+export const areBlobPathParts = (
+	pathParts: string[],
+): pathParts is ["", typeof blobManagerBasePath, string] =>
+	pathParts.length === 3 && pathParts[1] === blobManagerBasePath;
```

Comment:
This is more restrictive than the old check (used to do `length >= 2`). Any concerns with this being less future-proof, if we start having multi-level IDs for blobs? I guess it's likely all the code would need to be taught to read these paths before we could start writing them, so this doesn't have to account for it.